src/model/wallet.ts

```diff
@@ -545,7 +696,10 @@ export const createWallet = async (
   }
 };
 
-export const savePrivateKey = async (address, privateKey) => {
+export const savePrivateKey = async (
+  address: EthereumAddress,
+  privateKey: null | EthereumPrivateKey
```

Comment:
hm can this ever be null (based on usage, worst it'll be undefined?)

Reply:
it's null for readonly wallets.